tests/snapshots/tests.contrib.sanic.test_sanic_server.test_multiple_requests_sanic_http.json

```diff
@@ -65,9 +65,9 @@
     "start": 1643465627811584000
   },
      {
-       "name": "__main__.random_sleep",
+       "name": "__mp_main__.random_sleep",
        "service": "sanic",
-       "resource": "__main__.random_sleep",
+       "resource": "__mp_main__.random_sleep",
```

Comment:
sanic>=22.9.0 spawns a child process to run an application. This changes the operation and resource name of the span generated by [random_sleep()](https://github.com/DataDog/dd-trace-py/blob/284787771abaff5b34cb417511089a04ce2ade9a/tests/snapshots/tests.contrib.sanic.test_sanic_server.test_multiple_requests_sanic_http.json#L22). If we override  the default span name in `tracer.wrap()` we won't need the new snapshot variant.